tools/docs/requirements.txt

```diff
@@ -32,9 +32,9 @@ colorama==0.4.4 \
     --hash=sha256:5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b \
     --hash=sha256:9f47eda37229f68eee03b24b9748937c7dc3868f906e8ba69fbcbdd3bc5dc3e2
     # via -r tools/docs/requirements.txt
-docutils==0.17.1 \
-    --hash=sha256:cf316c8370a737a022b72b56874f6602acf974a37a9fba42ec2876387549fc61 \
-    --hash=sha256:686577d2e4c32380bb50cbb22f575ed742d58168cee37e99117a854bcd88f125
+docutils==0.16 \
```

Comment:
if this fixes the problem then defo lets land, but i would also like to check further whether upstream are aware/fixing

Reply:
OK sounds good. I don't have the bandwidth to do that, so do you want to take that part on? I would just like to get this "fixed" as the docs look pretty bad.